src/WikiService.Mappers/Db/DbRubricMapper.cs

```diff
@@ -0,0 +1,40 @@
+﻿using LT.DigitalOffice.Kernel.Extensions;
+using LT.DigitalOffice.WikiService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.WikiService.Models.Db;
+using LT.DigitalOffice.WikiService.Models.Dto.Requests;
+using Microsoft.AspNetCore.Http;
+using System;
+
+namespace LT.DigitalOffice.WikiService.Mappers.Db
+{
+  public class DbRubricMapper : IDbRubricMapper
+  {
+    private readonly IHttpContextAccessor _httpContextAccessor;
+
+    public DbRubricMapper(
+      IHttpContextAccessor httpContextAccessor)
+    {
+      _httpContextAccessor = httpContextAccessor;
+    }
+
+    public DbRubric Map(CreateRubricRequest request)
+    {
+      if (request == null)
+      {
+        return null;
+      }
+
+      Guid rubricId = Guid.NewGuid();
```

Comment:
do you need create the variable here?

Reply:
changed create the variable to new DbRubric in Id prop